src/test/java/GitDifferTest.java

```diff
@@ -0,0 +1,88 @@
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.variantsync.diffdetective.datasets.PatchDiffParseOptions;
+import org.variantsync.diffdetective.datasets.Repository;
+import org.variantsync.diffdetective.diff.git.CommitDiff;
+import org.variantsync.diffdetective.diff.git.DiffFilter;
+import org.variantsync.diffdetective.diff.git.PatchDiff;
+import org.variantsync.diffdetective.diff.result.DiffParseException;
+import org.variantsync.diffdetective.util.IO;
+import org.variantsync.diffdetective.variation.diff.DiffTree;
+import org.variantsync.diffdetective.variation.diff.parse.DiffTreeParseOptions;
+import org.variantsync.diffdetective.variation.diff.parse.DiffTreeParser;
+import org.variantsync.diffdetective.variation.diff.serialize.Format;
+import org.variantsync.diffdetective.variation.diff.serialize.LineGraphExporter;
+import org.variantsync.diffdetective.variation.diff.serialize.edgeformat.ChildOrderEdgeFormat;
+import org.variantsync.diffdetective.variation.diff.serialize.nodeformat.FullNodeFormat;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class GitDifferTest {
+    private final static Path testDir = Constants.RESOURCE_DIR.resolve("diffs").resolve("differ");
+    private final static Path REPOS_DIR = Path.of("src").resolve("test").resolve("resources").resolve("repos");
+
+    private static final String[] commitIDs = new String[] {"a032346092e47048becb36a7cb183b4739547370", "4a4c1db1b192e221d8e25460d6d1128d1bdd0c0d", "d7eeba1b94f3d0c0ebed3457eea8fa8537143348"};
+
+    public static Stream<String> testCommits() throws IOException {
+        return Arrays.stream(commitIDs);
+    }
+
+    @ParameterizedTest
+    @MethodSource("testCommits")
+    public void test(String basename) throws IOException, DiffParseException {
+        testCase(basename);
+    }
+
+    private static Repository repo() {
+        final Path marlinPath = Path.of(".")
```

Comment:
Name seems to be a copy and paste error. Please rename because there is nothing about marlin here.

Reply:
done